src/css/_docs.scss

```diff
@@ -46,11 +46,10 @@
     &-overlay {
       position: relative;
       margin: 0 auto;
-      @include mq(mobile) {
```

Comment:
It looks like the width 90% would just become the default, which then swaps out at wider breakpoints.